test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -180,16 +177,8 @@ export default class PerfCompare extends Component {
             }
         )
 
-        const baselineBuildTestInfo = await fetch( `/api/getTestInfoByBuildInfo?url=${this.state.selectedRuns.baselineID[0]}&buildName=${this.state.selectedRuns.baselineID[1]}&buildNum=${this.state.selectedRuns.baselineID[2]}`, {
-            method: 'get'
-        } );
-
-        const testBuildTestInfo = await fetch( `/api/getTestInfoByBuildInfo?url=${this.state.selectedRuns.testID[0]}&buildName=${this.state.selectedRuns.testID[1]}&buildNum=${this.state.selectedRuns.testID[2]}`, {
-            method: 'get'
-        } );
-
-        let baselineTestResultsJson = await baselineBuildTestInfo.json();
-        let testTestResultsJson = await testBuildTestInfo.json();
+        let baselineTestResultsJson = await fetchData( `/api/getTestInfoByBuildInfo?url=${this.state.selectedRuns.baselineID[0]}&buildName=${this.state.selectedRuns.baselineID[1]}&buildNum=${this.state.selectedRuns.baselineID[2]}`);
+        let testTestResultsJson = await fetchData( `/api/getTestInfoByBuildInfo?url=${this.state.selectedRuns.testID[0]}&buildName=${this.state.selectedRuns.testID[1]}&buildNum=${this.state.selectedRuns.testID[2]}`);
```

Comment:
I think the original code is wrong. `baselineTestResultsJson` and `testTestResultsJson` do not get changed. We should use `const` in this case.